LICENSE-README.md

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2023 See the [README.md](README.md) at the bottom for authors/contributors.
+
+Update the date at the top accordingly (see <https://www.gnu.org/licenses/gpl-howto.html.en#copyright-notice>).
+
+This software is a digital implementation of the "FüSim MANV"/"FueSim MANV" (Führungssimulation Massenanfall von Verletzen), a German simulation system for training emergency medical services leadership personnel on how to manage Mass Casualty Incidents.
+
+The software is licensed with the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+Exceptions apply to image files in the frontend/src/assets folder, which may be covered by and therefore distributed under different licenses.
+The individual license for each image can be found in [frontend/src/assets/image-sources.md](frontend/src/assets/image-sources.md). Files added to this project may only use this exception if these files serve the only purpose of being displayed and will only be parsed for the purpose of displaying them.
+Especially, you must not use this exception for source code files.
+Another exception exists for datasets. Datasets are files that include no new features and just different compositions allowed through the source code given template class. Datasets must always be in human readable formats (for example csv, JSON, txt). Datasets license must be mentioned in a CUSTOM-LICENSE file in the folder stating which file is meant and which gives information about the license and author. These datasets are not allowed to create any new features or enhance the software in any way, as to create elements that can be part of an export and are importable into the distributed source code (this software) under the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. Therefore any dataset must be able to be part of an export and must be able to be imported into this software that must be made available under the terms of GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+All exceptions can be part of another license chosen by the author, but using this software with such exceptions allows everyone for testing purposes to inspect these exceptions and test if they conform to the given restrictions (rules) given to them, for example can datasets be exported by a user and imported for testing purpose only or read with a program, to check they comply to the restrictions. For this the user that uses such datasets in a software must have the permission of the author to give the users of this software these rights.
```

Comment:
This paragraph should make it possible for people to use custom data sets, e.g. own types of patients or vehicles. Author is the one creating the data sets, could be more clear.
The exception with testing is meant so that other people can check if they just added datasets or if they changed smth which falls under AGPL. So that we or others can't be sued for copying and inspecting the code and datasets from a server or so, for testing purposes to check if a violation accured.
Don't know if this is necessary.
Do you get the idea?
Exceptions should always be able to be checkable, even by a third person (e.g. you or me, given the data from a person taking part in an online training course of X), if it violates this license.

Reply:
I can see why we would want to inspect the code, although permission and free access is granted by AGPL itself. 

However, I can't imagine anything we could learn from inspecting the *datasets* as all "added" functionality is by definition reflected in the code we and anyone can access anyways. However, this 'checking' clause reads like any dataset must be effectively publicly accessible, possibly with restricted permission use by others, but still accessible.

Few authors of datasets might want to use the software under this condition and again, I do not see the benefit of asserting anything about the datasets.

I suggest we define datasets more explicitly (e.g. What is `... allowed through the source code given template class` supposed to refer to?) and include them in the paragraph about images. They are just custom assets after all.

Reply:
KISS - Keep It Simple Stupid.
I don't see an advantage in adding a lot of legal text to the license.
1. It will be nearly impossible to enforce.
2. If we formulate it ourselves, it is probably not foolproof and would probably not hold up in a court.
3. If something is too complicated, people either ignore it or are put off by it and don't use this software.

Reply:
I'd suggest deleting line 15 and moving the `Especially, you must not use these exception for source code files.` below the dataset definition.

Reply:
Is this resolved?

Reply:
@Nils1729 Can this be marked as `resolved`?